flow/designs/asap7/mock-array-big/macro-placement.tcl

```diff
@@ -9,8 +9,8 @@ for {set row 0} {$row < $rows} {incr row} {
   for {set col 0} {$col < $cols} {incr col} {
     set inst [$block findInst [format "ces_%d_%d" $row $col]]
 
-    set x [expr int(($margin_x + ($ce_pitch_x / 2) + ($ce_pitch_x * $col)) * $units)]
-    set y [expr int(($margin_y + ($ce_pitch_y / 2) + ($ce_pitch_y * $row)) * $units)]
+    set x [expr int(int(($margin_x + ($ce_pitch_x / 2) + ($ce_pitch_x * $col)) / $placement_grid_x) * $placement_grid_x * $units)]
```

Comment:
@maliberty Isn't there a Tcl interface to OpenROAD that I can use to get the placement grid?

So far I've been trawling various files, finding a number that looks good and trying it out and if it works in detailed routing I conclude that it's good 'nuf to use around the house...